src/syscalls.h

```diff
@@ -12,7 +12,7 @@ See the file LICENSE for details.
 void debug( const char *str );
```

Comment:
As we grow the number of system calls here, I would like to adjust the names, both to make it clear that this is not Unix, and to group things by functional area.

Please call these system calls `process_run, process_wait, process_kill, process_reap` and then rename `getpid` -> `process_self` and `getppid` -> `process_parent`

Reply:
Yes, all the symbols involved should ine up.